### ISSUE TITLE
Missing ALT Text - HOL Step-by-Step and Student/Trainer WDS

### ISSUE THREAD

**Issue by dzaremba90** [Microsoft/MCW-Containers-and-DevOps#14]
Alt text is not appearing on the images in the HOL Step-by-Step & WDS Trainer/Student Guide.

**Comment by DawnmarieDesJardins** [comment]
Also missing alt-text on the Before the lab document

**Comment by michelebusta** [comment]
alt text all fixed
